doc/whats_new/v1.1.rst

```diff
@@ -311,6 +311,9 @@ Changelog
 - |Enhancement| Adds support to use pre-fit models with `cv="prefit"`
   in :class:`ensemble.StackingClassifier` and :class:`ensemble.StackingRegressor`.
   :pr:`16748` by :user:`Siqi He <siqi-he>` and :pr:`22215` by
+- |Enhancement| :class:`feature_selection.GenericUnivariateSelect` preserves
```

Comment:
It seems that there is an issue with the merging here.

Reply:
I've been seeing more weird merge issues lately in the whats new in other PRs. It could be related to https://github.com/scikit-learn/scikit-learn/issues/21516

Reply:
I'm not sure I understand what the problem is -- everything looks fine on my end?

Reply:
There is no problem. Looks fine now. GitHub interface was doing something weird with the diff?

Reply:
It was missing your username: https://github.com/scikit-learn/scikit-learn/pull/22215/commits/9d6c27ad16148bf91999cfa74cff1f6025c67874

Usually, it happens that GitHub complains about merge conflict. In this case, merging locally will not show any conflicts but `git` is actually messing up the merging as it did for this one. Basically, GitHub is right but I don't know why `git` is resolving this merge conflict on its own.

It only happens with the `changelog` generally.